card/spec/models/card/type_transition_spec.rb

```diff
@@ -132,7 +132,7 @@ def increment_count()   Card.count += 1  end
 describe Card, "type transition create callback" do
   before do
     Card::Auth.as_bot do
-      Card.create(:name=>'Basic+*type+*delete', :type=>'Pointer', :content=>"[[Anyone Signed in]]")
+      Card.create(name: 'Basic+*type+*delete', type: 'Pointer', content: "[[Anyone Signed in]]")
```

Comment:
Line is too long. [96/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.